app/build.gradle.kts

```diff
@@ -17,9 +17,9 @@ plugins {
     id("com.android.application")
     id("org.jetbrains.kotlin.android")
     id("droidconke.android.hilt")
-    id("com.google.gms.google-services")
-    id("com.google.firebase.crashlytics")
-    id("com.google.firebase.firebase-perf")
+    id("droidconke.android.application.firebase")
+    id("org.jlleitschuh.gradle.ktlint")
```

Comment:
@misshannah these were removed on #153 , you can remove the in your next PR

Reply:
I did already.